examples/templates/exposed-thing/src/base.ts

```diff
@@ -76,14 +80,14 @@ export class WotDevice {
         ).then((exposedThing) => {
             this.thing = exposedThing;
             this.td = exposedThing.getThingDescription();
-            this.add_properties();
-            this.add_actions();
-            this.add_events();
-            this.thing.expose();
+            this.addProperties();   // Initialize properties and add their handlers
```

Comment:
I would also propose renaming these methods. They were named like this from the pre 0.7.x times where each affordance was added via methods and not via the produce(). A possible name could be `addPropertyHandlers` but there is also initialization happening.